modules/admin-ui/src/main/resources/web/admin/apps/content-manager/js/app/create/NewContentDialog.ts

```diff
@@ -60,8 +65,17 @@ module app.create {
             });
 
             this.contentList = new app.create.NewContentDialogList();
+            this.contentList.addClass("new-content-types-list");
+
+            this.mostPopularBlock = new api.dom.DivEl("most-popular-content-types-container hidden");
```

Comment:
There is no need to additional container only for title. It could be moved in of MostPopularItemsList and initialized in constructor